### DISCUSSION TITLE
ShadowShare订阅链接导入失败

### DISCUSSION THREAD

**Discussion by heidongwan** [shadowsocks/shadowsocks-android#2983] [Q&A]
ShadowShare 导出的订阅链接无法导入，显示失败，请处理一下

**Comment by thjw81** [comment]
我使用shadow socks 的 Windows Android 客户端，试过所有订阅的可能，从来没有成功订阅。这真是太奇怪了，其他客户端能正常订阅的链接放到它里面也不行。肯定是软件有问题。

**Comment by hello-earth** [comment]
这个软件的订阅内容和其他的不一样，你需要使用json格式返回订阅内容

**Comment by hello-earth** [comment]
> 
如，更能多详细信息你把本地的一个配置导出文件自己看一下就知道了
[
  {
    "server": "123.121.com",
    "server_port": 12345,
    "password": "12363",
    "method": "chacha20-ietf-poly1305",
    "remarks": "",
    "route": "gfwlist",
    "remote_dns": "dns.google",
    "ipv6": false,
    "metered": false,
    "proxy_apps": {
      "enabled": true,
      "bypass": false,
      "android_list": [
        "com.google.android.gms",
        "com.google.android.gsf"
      ]
    },
    "udpdns": false
  }
]
